Telstar Logistics Parcel Delivery Solution/Models/City.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Telstar_Logistics_Parcel_Delivery_Solution.Models
+{
+    [Keyless]
+    public class City
+    {
```

Comment:
Is this one missing the "disabled" field or did we scrap that?

Reply:
I will add the Disabled field in another PR if thats fine :)

Reply:
That's great! 👍